### PR TITLE
Configure RSpec and add first tests

### LINKED ISSUE #9: Configure RSpec and add first tests

### PR DESCRIPTION
**Pull request by nicolas-brousse**
Closes #9

- [x] Shared example
- [x] Correct RSpec config
- [x] Cache gems in CI

### CODE REVIEW

File: app/components/view_component/form/base_component.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@ def before_render
 
       def combine_options!
         @options = (self.class.default_options.deep_dup || {}).deep_merge(options).tap do |opts|
-          opts[:class] = class_names(options[:class], html_class) if (html_class || options[:class]).present?
+          # opts[:class] = class_names(options[:class], html_class) if (html_class || options[:class]).present?
```
Comment (nicolas-brousse) [inline app/components/view_component/form/base_component.rb:41]: @Spone Not sure how to deal with this

**Review by nicolas-brousse — COMMENTED** [review]
@Spone fyi a downgrade Rails requirement to v6.0

File: app/components/view_component/form/base_component.rb (hunk 1 above)
Comment (Spone) [reply app/components/view_component/form/base_component.rb:41]: Let's copy https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/actionview/lib/action_view/helpers/tag_helper.rb#L329-L345 or https://github.com/komposable/klassnames/blob/master/lib/klassnames/classnames_helper.rb#L2-L19 to support Rails 6.0

Comment (nicolas-brousse) [reply app/components/view_component/form/base_component.rb:41]: We will see in #21 then.

**Review by Spone — APPROVED** [review]

**Pull request merged**